apertium-tur-uzb.tur-uzb.dix

```diff
@@ -656,6 +660,7 @@
 <e><p><l>fırsat<s n="n"/></l><r>fursat<s n="n"/></r></p></e>
 <e><p><l>futbol<s n="n"/></l><r>futbol<s n="n"/></r></p></e>
 <e><p><l>gece<s n="n"/></l><r>kecha<s n="n"/></r></p></e>
+<e><p><l>dün<s n="n"/></l><r>kecha<s n="n"/></r></p></e>
```

Comment:
This is really more of an adverbial use.  Can you think of a noun use where "kecha" means "yesterday"?

Reply:
My bad, changing it to an adverb.

Reply:
`<e><p><l>dün<s n="adv"/></l><r>kecha<s n="adv"/></r></p></e>`
Correct?

Reply:
Yes